pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
+
+buildGoModule rec {
+  pname = "emacsclient-commands";
+  version = "0.0.0-unstable-2023-09-22";
+
+  src = fetchFromGitHub {
+    owner = "szermatt";
+    repo = pname;
+    rev = "8f5c8a877794ed51f8225036e36fd5ce272b17f3";
+    hash = "sha256-OlcB5VqWYdl0wz1y8nmG6Xgdf5IPOUQ31UG1TDxQAis=";
+  };
+
+  vendorHash = "sha256-8oREed2Igz5UvUTDdOFwW5wQQy3H8Xj8epxo6gqnZFA=";
+
+  buildInputs = with pkgs; [go gnumake];
+
+  buildPhase = ''
+DESTDIR=$out/ make install
+'';
+
+  meta = with lib; {
+    description = "a collection of small shell utilities that connect to a local Emacs server";
+    homepage = "https://github.com/szermatt/emacsclient-commands";
+    license = licenses.mit;
```

```suggestion
    license = licenses.gpl2Only;
```
I don't know where you got the mit information from but the repo says gpl2
https://github.com/szermatt/emacsclient-commands/blob/master/license